uber/templates/registration/new.html

```diff
@@ -10,25 +10,14 @@
 <script src="../static/js/servertimecheck.js" type="text/javascript"></script>
 
 <script type="text/javascript">
-    // automatically log out after 15 minutes of inactivity
-    setTimeout(function () {
-        window.location = "../accounts/logout";
-    }, 900000);
-    
+    {% if not c.DEV_BOX %}
+        // automatically log out after 15 minutes of inactivity
+        setTimeout(function () {
+            window.location = "../accounts/logout";
+        }, 900000);
+    {% endif %}
```

Comment:
Being logged out every 15 minutes is indeed a good idea on-site, but it was really annoying on my local dev machine, so I turned that off when ``c.DEV_BOX`` is turned on.